src/Sql/SqlPgsql.php

```diff
@@ -170,4 +171,46 @@ public function getPasswordFile()
     {
         return $this->password_file;
     }
+
+    /**
+     * Get schema name.
+     *
+     * @return string
+     */
+    public function getSchemaName()
+    {
+        $schema = 'public';
+        $dbSpec = $this->getDbSpec();
```

Comment:
@RoSk0 Hello, this method changed to "db_spec()" in master. The patch does not apply anymore, could you take a look please ? :)